package.json

```
@@ -173,7 +173,8 @@
     "lint": "npm run lint-fmt && npm run lint-eslint",
     "lint-eslint": "eslint .",
     "lint-fmt": "prettier --check .",
-    "test": "node --test"
+    "test": "cd test && node --test && cd ..",
```

```suggestion
    "test": "node --test **/test/**/*.{cjs,mjs,js}",
```

you're change breaks `npm test --watch`